main.go

```diff
@@ -96,7 +101,7 @@ func main() {
 
 		setupLogger(config.LogLevel)
 
-		db, err := bolt.Open(fmt.Sprintf(".bolt.db.%d", config.Raft.RaftId), 0600, nil)
+		db, err := bolt.Open(fmt.Sprintf(c.String("work-dir")+".bolt.db.%d", config.Raft.RaftId), 0600, nil)
```

Comment:
is it possible move boltdb initialization statement within manager.Start method?